src/libcore/cell.rs

```diff
@@ -1245,6 +1245,38 @@ impl<'b, T: ?Sized> Ref<'b, T> {
         let borrow = orig.borrow.clone();
         (Ref { value: a, borrow }, Ref { value: b, borrow: orig.borrow })
     }
+
+    /// Convert into a reference to the underlying data.
+    ///
+    /// The underlying `RefCell` can never be mutably borrowed from again and will always appear
+    /// already immutably borrowed. It can still be immutably borrowed until more than `isize::MAX`
```

Comment:
I don't think we document that it's internally an `isize` anywhere else so we should probably avoid mentioning that here. I would suggest that we document it as it not being a good idea to leak more than a relatively small, constant amount of references.

But if we choose to leave this as an `isize`, then I would suggest noting that the specific maximum may change in the future.